custom_components/tahoma/tahoma_entity.py

```diff
@@ -70,7 +70,7 @@ def unique_id(self) -> str:
     @property
     def assumed_state(self) -> bool:
```

Comment:
What do you think to add this to all the properties as guard condition ? Or that’s useless ?

Reply:
@tetienne I am not sure if all properties need this, normally this would be handled by HA core already.

Perhaps is_closing and is_opening should also not be enabled for assumed state in core?